docs/Clearwater_Configuration_Options_Reference.md

```diff
@@ -130,7 +130,6 @@ This section describes optional configuration options, particularly for ensuring
 * `hss_mar_scheme_digest` - if Clearwater determines a subscriber is trying to use password-based digest authentication, this field determines what authentication scheme it requests in the Multimedia-Auth-Request. Default value is 'SIP Digest'.
 * `hss_mar_scheme_akav1` - if Clearwater determines a subscriber is trying to use AKAv1 authentication, this field determines what authentication scheme it requests in the Multimedia-Auth-Request. Default value is 'Digest-AKAv1-MD5'.
 * `hss_mar_scheme_akav2` - if Clearwater determines a subscriber is trying to use AKAv2 authentication, this field determines what authentication scheme it requests in the Multimedia-Auth-Request. Default value is 'Digest-AKAv2-SHA-256'.
-* `allow_fallback_ifcs` - when Clearwater cannot find an Identity element in the user profile XML from the HSS that matches the served user, it will by default not invoke any services. When this is set to Y, it will instead use the first set of Initial Filter Criteria in the profile (giving the ability to work around HSSes that return something unexpected, or the ability to get basic support for wildcard public service identities).
```

Comment:
Surely we should keep this, as we still need the first ability it gives us ("to work around HSSes that return something unexpected"), and we should just update it's explanation?